app/styles/app.css

```diff
@@ -132,10 +132,12 @@
 
 .diff-highlight .removed-line {
   background-color: rgba(255, 0, 0, 0.1);
+  user-select: none;
 }
 
 .dark .diff-highlight .removed-line {
   background-color: rgba(255, 0, 0, 0.15);
+  user-select: none;
```

Comment:
@andy1li you don't need this override btw, it should automatically apply based on the `.diff-highlight .removed-line` one above